### ISSUE TITLE
Support for tf.keras

### ISSUE THREAD

**Issue by mmesensol** [Prodicode/ann-visualizer#20]
Is it possible to add support for models created with tf.keras instead of keras?

Otherwise, do you know of a similar tool with such support?

Thanks

**Comment by Hemantr05** [comment]
Yes, It is possible to use tf.keras instead of keras if you're tensorflow
to develop a neural network instead of keras.

Just like the above, the layers and models can also be added ,
tf.keras.models , tf.keras.layers and so on.

**Comment by mmesensol** [comment]
Thanks. I was able to get it to work by hacking around the ann_visualizer/visualize.py file and replacing keras.* by tf.keras.* in different ways. It wasn't very straightforward, though.

Still, I got stuck because ANN Visualizer like that my model's first layer is a Flatten. Is there a way to make it work, or is such a limitation by design?

Thanks

**Comment by Hemantr05** [comment]
There is an another alternative of visualizing you neural network
architecture,

https://keras.io/visualization/
The above link is a part of the keras documentation to visualize the neural
network. I'm pretty sure, it can be access using tf.keras as well. You can
give it a try
